js/lightbeam.js

```diff
@@ -113,6 +113,16 @@ async function initLightBeam() {
   websites = await storeChild.getAll();
   renderGraph(websites);
 
+  const saveData = document.getElementById('save-data-button');
+  saveData.addEventListener('click', () => {
+    storeChild.getAll().then( data => {
```

Comment:
Also whilst you clean this up, you can clean the resetData then up which I did haha.